docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>nitpick: Some lines are too long, considering splitting line for a better reading experience</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>:+1:</p><p>proposed partitioning:</p><p>  images_in</p><p>  run_fn, setup_fn,</p><p>out_type, in_types,</p><p>outs_ndim, ins_ndim</p><p> </p>